environment.yml

```diff
@@ -11,35 +11,118 @@ channels:
 dependencies:
   - pip!=21.3
   - python>=3.8
-  # Python packages that cannot be installed from PyPI:
   - cartopy
-  - compilers
+  - cdo>=1.9.7
+  - cdsapi
+  - cf-units >=3.0.0,<3.1.0,!=3.0.1.post0  # see https://github.com/ESMValGroup/ESMValCore/issues/1655
+  - cftime
+  - cmocean
+  - cython
+  - dask
+  - ecmwf-api-client
+  - eofs
   - esmpy
-  - esmvalcore
+  - esmvalcore 2.5.*
```

Comment:
What about the newest version? And same for the osx environment file

Reply:
Would it be OK with you to do that in a new pull request? That seems a bigger change than what the intent of this pull request was (i.e. use setuptools-scm for the version number and install development dependencies from conda-forge).

Reply:
I *think* that means a minimum global pin, Saskia - mamba will of course grab 2.6 if all's good with the other deps

Reply:
@bouweandela so that's not a minimum pin? If not, let's go 2.6.0 already - actually - I am now confised as to how the pinning on Core is going on

Reply:
No, @valeriupredoi, that is not correct. If in doubt, please refer to [the package match specification](https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#package-match-specifications). In this case, this will pickup any 2.5 version. This is intended to allow for bugfix releases.

Reply:
righto - so we change it only when we have Tool 2.6 rolled out as well?

Reply:
I was under the impressions that a big chunk of the development has been on holidays for a very long time, but if you think this can be a source of issues I will open another pull request.